glue_jupyter/widgets/layer_options.py

```diff
@@ -69,6 +71,15 @@ def _update_layers_from_glue_state(*args):
             self.layers = [layer_to_dict(layer_artist, i) for i, layer_artist in
                            enumerate(self.viewer.layers)]
 
+        def _on_data_or_subset_update(msg):
+            if msg.attribute == 'label' or msg.attribute == 'style':
```

Comment:
Maybe shorter like this?
```suggestion
            if msg.attribute in ('label', 'style'):
```